Jenkinsfile_CNP

```diff
@@ -22,34 +22,19 @@ def vaultSecrets = [
     secret('test-subscription-key', 'TEST_SUBSCRIPTION_KEY'),
 
     secret('storage-account-secondary-key', 'TEST_STORAGE_ACCOUNT_KEY'),
-    secret('storage-account-name', 'TEST_STORAGE_ACCOUNT_NAME'),
```

Comment:
Changes in this file revert my PR from today. I guess this is by mistake.

Reply:
and more